### PR TITLE
Microsoft.VisualStudioCode_v1.69.0

### PR DESCRIPTION
**Pull request by lszomoru**
Microsoft.VisualStudioCode_v1.69.0

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65394)

### CODE REVIEW

**Review by jedieaston — APPROVED** [review]

File: manifests/m/Microsoft/VisualStudioCode/1.69.0/Microsoft.VisualStudioCode.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+InstallerSwitches:
+  Custom: /mergetasks=!runcode
+Installers:
+  - Architecture: arm64
+    Scope: machine
+    InstallerUrl: 'https://az764295.vo.msecnd.net/stable/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/VSCodeSetup-arm64-1.69.0.exe'
+    InstallerSha256: 9ab6f6c919e9e158ae5a7abcaf7d3f3ff1c9c6af76888d742749b2baeccbe3b5
+  - Architecture: arm64
+    Scope: user
+    InstallerUrl: 'https://az764295.vo.msecnd.net/stable/30d9c6cd9483b2cc586687151bcbcd635f373630/VSCodeUserSetup-arm64-1.68.1.exe'
```
Comment (jedieaston) [inline manifests/m/Microsoft/VisualStudioCode/1.69.0/Microsoft.VisualStudioCode.installer.yaml:15]: @lszomoru I missed this when I was reviewing, but can you make another PR that makes it so all of the links are for 1.69.0? Some of these are still for 1.68.1.

Comment (justinkb) [reply manifests/m/Microsoft/VisualStudioCode/1.69.0/Microsoft.VisualStudioCode.installer.yaml:15]: https://github.com/microsoft/winget-pkgs/pull/65441
